content/admin/org_settings/ServerDefaults.md

```diff
@@ -29,41 +23,13 @@ The General section of the Servers page includes a checkbox to **Enable automati
-
-### Sampling
-Sampling is a mode of operation in Contrast that greatly reduces the runtime performance penalty of the security analysis. It reduces the performance by eliminating redundant analysis on the same URLs over and over. The idea is that once a URL has been properly *sampled*, you can stop monitoring during requests for that URL.
-
-You can enable sampling while downloading an engine in the **Download the Engine** step of the installation wizard. To do this, perform the following steps:
-
-* Select **I want to configure a proxy or customize my settings**
-* Select the **Sampling** tab and check the **Use Sampling** checkbox
-
-At this point, you can input **Baseline**, **Frequency** and **Window** values.
-
-#### Setting sampling values
-The default values provided will help a lot, but you may want to configure these values more precisely.
-
-* **Baseline:** This is how many times a URL needs to be analyzed before it is considered *sampled*. Once a URL has been hit this number of times, Contrast will be *turned off* during future requests for that URL, which will allow them to be processed a lot faster. 
-* **Frequency:** After the **Baseline** samples of a URL have been taken, only every **N**th request will be analyzed fully. Frequency is that **N** value.
-* **Window:** After this many seconds passes, the *sampling window* will be reset and the **Baseline** samples will be drawn again.
-
-#### Enabling sampling via system properties (Java agent)
-You can also enable sampling by passing in the ```-Dcontrast.sampling``` JVM System property. Passing an empty system property will enable sampling with a:
-
->**Frequency** of 5
->**Baseline** of 5
->**Window** of 180 
-
-Alternatively, you can use ```-Dcontrast.sampling``` = X, Y, Z where: 
+Assessment provides detailed information on vulnerabilities discovered by Contrast so that you may track, share and receive remediation guidance. Turning Assessment on allows you to enable sampling and designate how stacktraces are captured. 
 
->X = **Baseline**
->Y = **Frequency** 
->Z = **Window**
+For more information on sampling and stacktraces, go to the [Server Settings](user-servers.html#settings) article.
 
 ## Protection Options
-Protection provides monitoring of your servers and applications - identifying and blocking attacks in real time. Turning Protection on gives you the option to bot block, which allows Contrast to use simple signaturing to block traffic from scrapers, attack tools and other unwanted automation.
+Protection provides monitoring of your servers and applications - identifying and blocking attacks in real time. Turning Protection on gives you the option to bot block, which allows Contrast to use simple signaturing to block traffic from scrapers, attack tools and other unwanted automation. 
```

Comment:
Regarding, "Turning Protection on gives you the option to bot block..."

Turning on protection also enables Syslog. It is mentioned in the next section but it may be good to mention that its tied to protect along with bot blocking.